ubuntudesign/vanilla-framework
# Update table styling

## Done
- Updated the table styles
- Created text-align utility component.

## QA
- Check this branch out in vf.io
- Go to http://localhost:3000/base/table/
- Check it looks good and rows with numbers are right aligned
- Go to http://localhost:3000/utilities/text-align/
- Check that is also ok

## Details
- Issue: https://github.com/ubuntudesign/vanilla-framework/issues/536
